projects/01-pair-matching/Mahya-Behboudi/1

```diff
@@ -0,0 +1,124 @@
+const selectors = {
```

Comment:
Is this file used in the project? If not please remove it.
And if it is please rename it to some meaningful name + .js